src/main/scala/com/yotpo/metorikku/output/writers/kafka/KafkaOutputWriter.scala

```diff
@@ -56,12 +61,14 @@ class KafkaOutputWriter(props: Map[String, String], config: Option[Kafka]) exten
         log.info(s"Writing Dataframe to Kafka Topic ${kafkaOptions.topic}")
         val df: DataFrame = selectedColumnsDataframe(dataFrame)
         val stream = df.writeStream.format("kafka")
+          .trigger(Trigger.ProcessingTime(kafkaOptions.processingTime))
```

Comment:
Why not allow full control of the trigger? Why force a specific trigger.
Also let's add support for watermarking if we're doing this fix